corehq/apps/export/static/export/js/download_export.ng.js

```diff
@@ -140,7 +140,20 @@
                 $('form[name="exportFiltersForm"]'));
             $scope.prepareExportError = null;
             $scope.preparingExport = true;
-            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {"Export type": $scope.exportList[0].export_type});
+            var userTypes = hqImport("hqwebapp/js/initial_page_data").get('user_types');
+            function getFilterName(exportType) {
+                return (exportType === "form" ? "emw" : "case_list_filter");
+            }
+            hqImport('analytix/js/kissmetrix').track.event("Clicked Prepare Export", {
+                "Export type": $scope.exportList[0].export_type,
+                "filters": _.map(
+                    $("#exportFiltersFormId input[name=" + getFilterName($scope.exportList[0].export_type) + "]")
+                        .serializeArray()[0].value.split(','),
+                    function (item) {
+                        if (item[0] === "t") { return userTypes[item.substring(3)]; }
+                        else { return item; }
```

Comment:
nit: don't need `else` when the `if` clause returns

Reply:
Cool, didn't know that https://github.com/dimagi/commcare-hq/pull/22084/commits/9c703d558439f9a80b81879c7062bc7fb607cacc

Reply:
Oh, I meant that as a flow control/logical thing that makes indentation look clearer (to me at least), not as a javascript thing.

Reply:
Oh yeah haha. That somehow seemed js specific at the time, maybe I was still in the zone of assuming everything here would be foreign to me.